packages/generators/generators/lib/plops/api.js

```diff
@@ -90,18 +97,41 @@ module.exports = plop => {
         return baseActions;
       }
 
-      const routeType =
-        answers.kind === 'singleType'
-          ? 'single-type-routes.js.hbs'
-          : 'collection-type-routes.js.hbs';
+      if (!answers.createContentType) {
+        return [
+          {
+            type: 'add',
+            path: `${filePath}/routes/{{id}}.js`,
+            templateFile: `templates/single-route.js.hbs`,
+          },
+          ...baseActions,
+        ];
+      }
+
+      if (answers.generateDefaultRoutes) {
```

Comment:
I'm wondering about the consistency to not create routes but still create a controller or a service 🤔  It doesn't feel super clear to me

Reply:
It's a change that @petersg83 proposed. But I might have put it in the wrong generator (in the api instead of the CT).

Reply:
not necessarily but I see two scenarios:

Create a simple api with no CT = a dumb controller / a dumb service / a dumb router.
Create an api with a CT = a CT / the associated Routes / the associated controller / the associated service.

In my factories PR I need to create new controllers in the generator so maybe we can merge this one and I'll add the rest in mine.

Reply:
If we always need routes when generating an API and CT then I'll remove the question as it doesn't makes sense.

However, do we need to generate routes in case we only create a CT? Because I don't think we have that explicitly taken care of.

Reply:
The answer isn't easy especially with the work I'm doing right now on the other PR. would be better to do a quick workshop :D